lsp-mode.el

```diff
@@ -1924,7 +1930,7 @@ WORKSPACE is the workspace that contains the progress token."
                                         'display `((height 0.9))
                                         'face lsp-modeline-code-actions-face)))))
     (propertize string
-                'help-echo (concat "Apply code actions (s-l a a)\nmouse-1: "
+                'help-echo (concat (format "Apply code actions (%s a a)\nmouse-1: " lsp-keymap-prefix)
```

Comment:
You may want to use a combination of `where-is-internal` and `key-description` to automatically get the key bind.
Something like 
```suggestion
                'help-echo (concat (format "Apply code actions (%s)\nmouse-1: " )
                                   (->> 'lsp-execute-code-action
                                        (where-is-internal)
                                        (-find (lambda (o) (not (member (aref o 0) '(menu-bar normal-state)))))
                                        (key-description)))
```

Reply:
Good idea :)